tests/Dockerfile

```diff
@@ -1,38 +1,32 @@
-FROM rust:latest
+FROM rust
+
+# Install Tarpaulin (code coverage tool)
+# https://github.com/xd009642/tarpaulin
+COPY --from=xd009642/tarpaulin /usr/local/cargo/bin/cargo-tarpaulin /usr/local/cargo/bin/cargo-tarpaulin
```

Comment:
We copy in the binary directly from the built docker image of tarpaulin.
This seems much easier and faster than building it manually. 😄